src/help.ts

```diff
@@ -6,6 +6,10 @@ export abstract class BBHelpClient {
   private static defaultHelpKey: string = 'default.html';
   private static currentHelpKey: string;
 
+  private static widgetLoaded: boolean = false;
+  private static DURATION: number = 100;
+  private static MAX_ITERATIONS: number = 100;
```

Comment:
I learned this from someone else, but it's preferred to put static variables outside of your classes so that they're not bundled with their "long" names. For example, you could add the `DURATION` variable outside the class and it would be perfectly acceptable:

```
const DURATION = 100;
```

This will also keep your class definitions cleaner.

Another thought: in the case of "duration" and "max iterations", those particular variables could probably be included inside their respective functions, since they're not being used elsewhere. This buys you slightly more readable code.

```
public static ready(): Promise<any> {
  const duration = 100;
  const maxIterations = 100;

  return new Promise((resolve: any, reject: any) => {
    // ... 
    const interval: any = setInterval(() => {
      // ...

      if (readyAttempts >= maxIterations) {
        // ...
      }
    }, duration);
  });
}
```